### ISSUE TITLE
Get Token from a newly create DeployToken

### ISSUE THREAD

**Issue by kingmesal** [jdalrymple/gitbeaker#3655]
**Description**

I've checked the api docs: https://docs.gitlab.com/ee/api/deploy_tokens.html#create-a-project-deploy-token and it shows the create project deploy token returns:
```json
{
  "id": 1,
  "name": "My deploy token",
  "username": "custom-user",
  "expires_at": "2021-01-01T00:00:00.000Z",
  "token": "jMRvtPNxrn3crTAGukpZ",
  "revoked": false,
  "expired": false,
  "scopes": [
    "read_repository"
  ]
}
```

This api: `api.DeployTokens.create(name, scopes, { projectId, expires_at: expiresAt });`
Returns:
```typescript 
interface DeployTokenSchema extends Record<string, unknown> {
    id: number;
    name: string;
    username: string;
    expires_at: string;
    revoked: boolean;
    expired: boolean;
    scopes?: DeployTokenScope[];
}
```

Where is the `token` field?

Side note, I noticed that this API was just recently expanded, so thanks for that!

**Comment by jdalrymple** [comment]
Probably just missed it in the type! EZ fix

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`42.0.1`](https://github.com/jdalrymple/gitbeaker/releases/tag/42.0.1) :rocket:
